### PR TITLE
Add snap install instructions

### PR DESCRIPTION
**Pull request by flexiondotorg**
Wordpress Desktop is now published in the Snap store, this pull request add instructions for installing the snap. Wordpress Desktop is currently in the beta channel but when it's promoted to stable I will file a another pull request to update the documentation.

### CODE REVIEW

**Comment by johngodley** [comment]
Thanks! Can you tell me how the WP Desktop app is built for Snap? Is it a copy of the download available from [here](https://apps.wordpress.com/desktop/)?

**Comment by flexiondotorg** [comment]
@johngodley Yes, it basically takes the deb you publish and transforms it. Here is the `snapcraft.yaml`

  * http://bazaar.launchpad.net/~flexiondotorg/+junk/snap-wordpress-desktop/view/head:/snap/snapcraft.yaml

That said it could be built directly from your source and integrated into CI. I'd be happy to give you some Snapcraft orientation so you could take ownership of publishing the snap directly. You'd then be in full control of the publishing/update process and have access to install metrics via the Snapcraft store.

File: docs/install.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ To be able to clone the repo and run the application you need:
 
 ## Installing and Running
 
+### Ubuntu and [snap supported distros](https://snapcraft.io/docs/core/install)
```
Comment (spen) [inline docs/install.md]: Might it be better to have this set of instructions follow the `From Source` instructions.
I would imagine the existing instructions would serve more folks and as such should take some precedence in the documents hierarchy.

Any thoughts? cc: @johngodley @astralbodies?

Comment (astralbodies) [inline docs/install.md]: Having the section last makes sense - and I'd like to see it under a new heading to point out it's a pre-packaged/compiled offering. Also please point out in the instructions that we're not officially supporting the distribution channel (until we decide to publish the releases ourselves). So something like:

---

### Pre-Packaged Distributions

#### Official Direct Downloads

Install the officially released desktop app by visiting [https://apps.wordpress.com/desktop/](https://apps.wordpress.com/desktop/). Download the archive for your platform of choice and install manually.

#### Unofficial Linux (Ubuntu & others) via [snap](https://snapcraft.io/docs/core/install)
...

---

Please update your PR to create this new section, replace the heading for snap, and put this as the last part of the readme. Thanks!

**Comment by astralbodies** [comment]
Closing for now; please reopen after comments addressed!

**Pull request closed without merging**